gprofiler/utils/process.py

```diff
@@ -15,14 +15,16 @@ def ensure_running(process: Process) -> None:
 
 
 def process_comm(process: Process) -> str:
-    try:
+    '''try:
         status = Path(f"/proc/{process.pid}/status").read_text()
     except FileNotFoundError:
         raise NoSuchProcess(process.pid)
     else:
         # ensures we read the right comm (i.e PID was not reused)
-        ensure_running(process)
+        ensure_running(process)'''
+    ensure_running(process)
+    return process.name()
 
-    name_line = status.splitlines()[0]
+    '''name_line = status.splitlines()[0]
     assert name_line.startswith("Name:\t")
-    return name_line.split("\t", 1)[1]
+    return name_line.split("\t", 1)[1]'''
```

Comment:
Instead - you can provide this implementation of the function under `if is_windows()`, and provide a Windows alternative that does just `process.name()`